vertx-web-client/src/main/java/io/vertx/ext/web/client/HttpResponse.java

```diff
@@ -119,20 +120,29 @@
    */
   @CacheReturn
   @Nullable
-  String bodyAsString();
+  default String bodyAsString() {
```

Comment:
Can you clarify @pavolloffay ? It seems to me this method should also get a default impl.